packages/cli/src/service/pipeline.ts

```diff
@@ -113,12 +111,12 @@ export async function install(filename: string, opts: any): Promise<void> {
     // check the installation
     logger.info('check plugin:');
     let isSuccess = true;
-    await constants.PLUGINS.map(async (plugin) => {
+    for (const plugin of constants.PLUGINS) {
```

Comment:
This will remove the concurrent running, do we have any specific reason for that?

Reply:
The plugin project can't run concurrently, and the `await` can't take effect.

Reply:
The original intent here was to install the plugin serial.

Reply:
Questions: a) What do you mean by the "plugin project" b) why does the `await` not taking effect?

Reply:
a) "plugin project" means the plugin nodejs project, we can't install package concurrently by mutil-process.
b) we can't await a map funtion.
test code:
```js
const arr = [1, 2, 3];
async function test() {
  await arr.map(async n => {
    setTimeout(() => console.log(n), 100);
  })
  console.log(4);
}

test();

```
output:
```shell
4
1
2
3
```

Reply:
Question A may be not a problem after #455 , I'll test it.